experimental/federated_plugin/federated_plugin/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: federated_plugin
```

Comment:
There's a line in the example app's pubspec.yaml for `publish_to: none`. That should be added here as well -- it'll keep anyone from accidentally publishing the package to pub.dev.